package.json

```diff
@@ -58,7 +58,7 @@
   "dependencies": {
     "@types/lodash": "^4.14.191",
     "chevrotain": "^10.4.2",
-    "langium": "^1.0.0",
+    "langium": "^1.1.0",
```

Comment:
We should use `~1.1.0` here and for `langium-cli` to prevent downstream projects to automatically pull higher minor versions.